modules/nw-sriov-networknodepolicy-object.adoc

```diff
@@ -50,6 +50,11 @@ When specifying a name, be sure to use the accepted syntax expression `^[a-zA-Z0
 <5> Optional: The priority is an integer value between `0` and `99`. A smaller value receives higher priority. For example, a priority of `10` is a higher priority than `99`. The default value is `99`.
 
 <6> Optional: The maximum transmission unit (MTU) of the virtual function. The maximum MTU value can vary for different network interface controller (NIC) models.
++
+[IMPORTANT]
+====
+Ensure that the MTU is not set to a value that does not match the cluster MTU.
```

Comment:
```suggestion
Ensure that the MTU is not set to a value that does not match the cluster MTU if you want to create VFs on default network interface
```

Reply:
@zhaozhanqi - Can you please check if the fix looks fine now.